### PR TITLE
protos: add protos for CreateFeedback API

### PR DESCRIPTION
**Pull request by scarlettperry**
### Description
PR adds protos for creating a feedback submission.

Context w.r.t to the `id` in request/response and `user_submitted`:

In one of the team discussions around implementation, a suggestion raised was to send a feedback request each time a user selects a rating option and to update the persisted feedback record with the latest feedback submission. This is why the request sends a unique client generated feedback id so that after the first feedback request is made, we can continue updating the feedback entry at that id. 

The `user_submitted` field is to indicate whether the user formally submitted feedback (by clicking the "submit" button) since we're creating/updating feedback submissions before a user formally submits it.

We may ultimately not go with this implementation approach if we think non formally user-submitted feedback won't be helpful data but I added support for it in the protos for now.

### GitHub Issue
https://github.com/lyft/clutch/issues/1797

### CODE REVIEW

**Comment by danielhochman** [comment]
how about using a client generated ID instead of a server/database generated ID?

**Comment by scarlettperry** [comment]
> how about using a client generated ID instead of a server/database generated ID?

thanks for the suggestion! hadn't thought to do it this way. did some research to understand the pros/cons. just want to confirm if these are the pros you were thinking of and if there are others you had in mind.
* not leaking persistence-specific concerns to client
* security reasons. not exposing DB primary keys
* assuming DB can create auto-incrementing ids (not an issue for us but in case users adopt a different DB or we change ours in the future)

as for the cons, the main one is trusting that the client-generated id will always be unique. i know the likelihood is very small and there are packages for this, ex: https://www.npmjs.com/package/uuid.

so with this approach, we'd have a unique constraint on the client id and a regular auto-incrementing DB id as primary key

**Review by mikecutalo — APPROVED** [review]

File: api/feedback/v1/feedback.proto [hunk 1]
```diff
@@ -58,3 +66,44 @@ message GetSurveysResponse {
   // the key will be the feedback origin name
   map<string, Survey> origin_survey = 1;
 }
+
+message FeedbackMetadata {
+  // extra info on the feedback (i.e. the survey question, the rating options, the feedback origin, etc.)
+  string origin = 1 [ (validate.rules).string = {min_bytes : 1} ];
```
[the remark this answers is not in the document]
Comment (scarlettperry) [reply api/feedback/v1/feedback.proto:72]: ah yea, will update

[the remark this answers is not in the document]
Comment (scarlettperry) [reply api/feedback/v1/feedback.proto]: updated

**Review by danielhochman — APPROVED** [review]

**Pull request merged**